### ISSUE TITLE
dutch locale shown on a fr/en install

### ISSUE THREAD

**Issue by jqueuniet** [Depressurizer/Depressurizer#55]
**Describe the bug**
After updating to 3.0.0, the interface is now using a dutch locale for most of the text. Some parts are still written in english.

**To Reproduce**
1. Start Depressurizer

**Expected behavior**
A relevant locale should be used, or English as a default

**Desktop:**
 - OS: Windows 10 1803, french locale

**Comment by shannonr** [comment]
I'm having the same issue on an en/en install. Even after setting Depressurizer to English and restarting there is still Dutch in the UI.

I updated an earlier version (sorry I didn't check what version I was on before I upgraded) by copying the contents of the zip over the old installation. 

I removed the settings folder from C:\Users\Shannon\AppData\Roaming and ran Depressurizer again and it works like it should with English everywhere, so the problem must be with upgrading from an earlier version.

OS: Windows 10 1803, English
